src/components/tab-panel/examples/tab-panel.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, h, State } from '@stencil/core';
+import { Tab } from '@limetech/lime-elements';
+
+@Component({
+    tag: 'limel-example-tab-panel',
+    shadow: true,
+    styleUrl: 'tab-panel.scss',
+})
+export class TabPanelExample {
+    @State()
+    private tabs: Tab[] = [
+        {
+            id: 'joker',
+            text: 'Joker',
+            icon: 'joker',
+            active: true,
+            iconColor: 'var(--lime-green)',
+        },
+        {
+            id: 'parasite',
+            text: 'Parasite',
+            icon: 'insect',
+            iconColor: 'var(--lime-magenta)',
+        },
+        {
+            id: 'harriet',
+            text: 'Harriet',
+            icon: 'administrator_female',
+            iconColor: 'var(--lime-orange)',
+        },
+    ];
+
+    constructor() {
+        this.handleChangeTab = this.handleChangeTab.bind(this);
+    }
+
+    public render() {
+        return [
+            <limel-tab-panel
+                tabs={this.tabs}
+                onChangeTab={this.handleChangeTab}
+            >
+                <limel-example-tab-panel-content id="joker" />
```

Comment:
What do you think we loop through the tabs array and the id of this tag`limel-example-tab-panel-content` becomes dynamic.

Reply:
Yeah maybe :thinking: I think the most important part is to make the example intuitive and easy to understand, and I'm not sure which is better in that sense
What do you think @adrianschmidt ? Hard code it like this or do it in a more dynamic way for the example?

Reply:
It's probably better to keep it simple. Anyone creating tabs from an array ought to know how to convert this into a loop, and if not, it's probably a good thing that they'll have to ask someone. But for anyone not well acquainted with loops, and whose only goal is to create three static tabs, the current example is going to be much more helpful than one with a loop.

Reply:
I am not sure if I understand it correctly but who is going to use lime-elements/this example (I am wondering to know our audience if this person is going to have a difficulty to understand basic javascript operation such as `map`)? We have implemented loops in our code as I see it in other examples, for example in Button component. 

I get the point It would be simple to understand the code if we don't use loop. When I see the code, I felt it is a redundant code.